content/machinelearning/naive_bayesian_classifier.Jmd

```diff
@@ -60,15 +60,15 @@ first(seeds, 3)
 The task in which we are interested is to predict the cultivar to which a seed
 belongs, by using Naive Bayes classification to distribute their morphological
 features in different classes. At its core, Naive Bayes classification assumes
-that the probability of belonging to a class based on a single measurement
-depends on the probability of observing this measurement knowing the
-distribution for the class, and that different measurements are independent. The
+that the probability of belonging to a class based on a single measurement of the features
+depends on the probability of observing this measurement knowing the features'
+distributions for the class. It also assumes that different measurements are independent. The
 [Wikipedia page][nbcwp] has a very clear introduction to the theory, and we will
 adhere to its notation here.
 
-In short, the probability that a sample defined by an array $\mathbf{x}$ of
-observations of multiple variables belongs to class $C_k$ is *proportional* to
-$p(C_k) \prod p(x_i | C_k)$. Further, we can get the class to which the sample
+In short, given an array $\mathbf{x}$ of observations of multiple features,
+the probability ($p(C_k | x)$) that this sample belongs to the class $C_k$ is *proportional*
+to $p(C_k) \prod p(x_i | C_k)$. Further, we can get the class to which the sample
 should be assigned using $\hat y = \text{argmax}_{k \in K} p(C_k) \prod p(x_i |
 C_k)$ - the class that is the most probable is taken as the guess.
```

Comment:
I found this a bit confusing as an intro to NBC. I'm not sure how to reword it, but here's an attempt.